encryption.go

```diff
@@ -0,0 +1,413 @@
+package winrm
+
+import (
+	"bytes"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"regexp"
+	"strconv"
+	"strings"
+
+	ntlmssp "github.com/CalypsoSys/bobntlmssp"
+	ntlmhttp "github.com/CalypsoSys/bobntlmssp/http"
+	"github.com/masterzen/winrm/soap"
+)
+
+type Encryption struct {
+	ntlm           *ClientNTLM
+	SIXTEN_KB      int
+	MIME_BOUNDARY  []byte

```

Comment:
It doesn't seem very golang to have fields in uppercase, what's the rationale behind this?

Reply:
removed, now using const values